src/otx/core/data/entity/base.py

```diff
@@ -206,6 +207,143 @@ def __repr__(self) -> str:
         )
 
 
+class VideoInfo(tv_tensors.TVTensor):
+    """Meta info for image.
+
+    Attributes:
+        img_id: Image id
+        img_shape: Image shape (heigth, width) after preprocessing
+        ori_shape: Image shape (heigth, width) right after loading it
+        padding: Number of pixels to pad all borders (left, top, right, bottom)
+        scale_factor: Scale factor (height, width) if the image is resized during preprocessing.
+            Default value is `(1.0, 1.0)` when there is no resizing. However, if the image is cropped,
+            it will lose the scaling information and be `None`.
+        normalized: If true, this image is normalized with `norm_mean` and `norm_std`
+        norm_mean: Mean vector used to normalize this image
+        norm_std: Standard deviation vector used to normalize this image
+        image_color_channel: Color channel type of this image, RGB or BGR.
+        ignored_labels: Label that should be ignored in this image. Default to None.
+    """
+
+    clip_len: int
+    num_clips: int
+    frame_interval: int
+    video_reader: decord.VideoReader
+    avg_fps: float
+    num_frames: int
+    start_index: int
+    frame_inds: np.ndarray
+
+    @classmethod
+    def _wrap(
+        cls,
+        dummy_tensor: Tensor,
+        *,
+        clip_len: int = 8,
+        num_clips: int = 1,
+        frame_interval: int = 4,
+        video_reader: decord.VideoReader = None,
+        avg_fps: float = 30.0,
+        num_frames: int | None = None,
+        start_index: int = 0,
+        frame_inds: np.ndarray | None = None,
+    ) -> ImageInfo:
```

Comment:
```suggestion
    ) -> VideoInfo:
```
It seems typo.

Reply:
Fixed at b92e52520aed9a19150f81e80ea272b1e4135155